pkg/destroy/azure/azure.go

```diff
@@ -15,7 +16,7 @@ import (
 	"github.com/Azure/go-autorest/autorest"
 	azureenv "github.com/Azure/go-autorest/autorest/azure"
 	"github.com/Azure/go-autorest/autorest/to"
-	"github.com/pkg/errors"
+	perrors "github.com/pkg/errors"
```

Comment:
`github.com/pkg/errors` also implements the `As` functions like `errors` so we can just use this import and drop the std lib one. We do that in most places...